lib/kasket/relation_mixin.rb

```diff
@@ -2,7 +2,11 @@ module Kasket
   module RelationMixin
     def to_kasket_query(binds = nil)
       if arel.is_a?(Arel::SelectManager)
-        arel.to_kasket_query(klass, (binds || bind_values))
+       if ActiveRecord::VERSION::MAJOR < 5
+         arel.to_kasket_query(klass, (binds || bind_values))
+       else
+         arel.to_kasket_query(klass, (@values[:where].binds.map(&:value_for_database) + Array(@values[:limit])))
```

Comment:
OK so regarding this line. Kasket actually doesn't support [with](https://github.com/zendesk/kasket/blob/master/lib/kasket/visitor.rb#L29), [offset](https://github.com/zendesk/kasket/blob/master/lib/kasket/visitor.rb#L30), [order](https://github.com/zendesk/kasket/blob/master/lib/kasket/visitor.rb#L32), [group by](https://github.com/zendesk/kasket/blob/master/lib/kasket/visitor.rb#L47) or [having](https://github.com/zendesk/kasket/blob/master/lib/kasket/visitor.rb#L48).

So it's not worth looking for anything else than some binding in `where` or the `limit`. So despite being awful, this line looks actually to be right.